### DISCUSSION TITLE
assuring total user data loss after every page/iframe reload

### DISCUSSION THREAD

**Discussion by laynr** [jupyterlite/jupyterlite#743] [Q&A]
Trying to assure total user data loss after every page/iframe reload by following the [docs](https://jupyterlite.readthedocs.io/en/latest/howto/configure/storage.html#volatile-memory-storage) 

I created an overrides.json and tired putting it both the root of the web directory and under lab/ directory consisting of
```
{
  "jupyter-config-data": {
    "enableMemoryStorage": true,
    "contentsStorageDrivers":["memoryStorageDriver"],
    "settingsStorageDrivers":["memoryStorageDriver"]
  }
}
```
I tested by creating and saving a notebook and then reloading the page.  The notebook persisted and I expected that it would not.  Any help is appreciated.

**Comment by jtpio** [comment]
Thanks @laynr.

> I created an overrides.json

Normally these should be in `jupyter-lite.json` file instead of `overrides.json`. So maybe renaming the file will help.
